docs/essentials/preferences.md

```diff
@@ -55,7 +55,7 @@ To remove all preferences:
 Preferences.Clear();
 ```
 
-In addition to these methods each take in an optional `sharedName` that can be used to create additional containers for preference. Read the platform implementation specifics below.
+The above methods also take in an optional `sharedName` that can be used to create additional containers for preferences. Read the platform implementation specifics below.
```

Comment:
also takes

Reply:
@jamesmontemagno 

I'm not sure if it should be _also takes_.

to me _also takes_ would be used if I was talking about just one of the methods e.g.
>The above method _also takes_ in an optional sharedName

however, saying 
>The above methods _also takes_ in an optional sharedName

Sounds unusual as since Methods is a plural, it should use the singular _take in_